### PR TITLE
ZBUG-704 Updating to latest timezone

### PR DESCRIPTION
**Pull request by rupalid**
Updating to latest timezone info https://data.iana.org/time-zones/releases/tzdata2019b.tar.gz

Updated windows names
Updated extra-data
Updated timezone.ics
Updated TzMsg.properties

Need to update the translation in another TzMsg.properties

### CODE REVIEW

File: conf/timezones.ics [hunk 1]
```diff
@@ -3832,6 +3765,26 @@ RRULE:FREQ=YEARLY;WKST=MO;INTERVAL=1;BYMONTH=3;BYDAY=-1SU
 END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VTIMEZONE
+TZID:Europe/Dublin
```
Comment (fatkudu) [inline conf/timezones.ics:3766]: Note I have seen a lot of discussion about the new Dublin timezone.  It probably needs special testing.  Suggest scouting out the tz mailing lists for possible issues.  The main thing is that STANDARD time is now in summer and DAYLIGHT is now in winter.

Comment (rupalid) [reply conf/timezones.ics:3766]: Sure will do that

File: conf/tz/extra-data [hunk 2]
```diff
@@ -166,3 +170,6 @@ ZoneMatchScore	Asia/Tokyo		200
 ZoneMatchScore	Australia/Sydney	200
 ZoneMatchScore	Australia/Brisbane	200
 ZoneMatchScore	Africa/Harare		200
+ZoneMatchScore	America/Anchorage		200
```
Comment (jrjena136) [inline conf/tz/extra-data]: Indentation is not proper in this case

File: conf/tz/tools/WindowsTimeZoneInfo.txt [hunk 3]
```diff
@@ -1721,7 +1819,7 @@ ID: Belarus Standard Time
       Begins at 02:00 on Sunday of week 5 of March
       Ends at 00:00 on Saturday of week 1 of January
 ID: Russian Standard Time
-   Display Name: (UTC+03:00) Moscow, St. Petersburg, Volgograd
+   Display Name:       (UTC+03:00) Moscow, St. Petersburg
```
Comment (jrjena136) [inline conf/tz/tools/WindowsTimeZoneInfo.txt:1822]: remove the unnecessary tabs

Comment (rupalid) [reply conf/tz/tools/WindowsTimeZoneInfo.txt:1822]: This is a auto generated file.

**Review by jrjena136 — APPROVED** [review]

**Review by sneha-patil-synacor — APPROVED** [review]

**Pull request merged**